.github/workflows/check-acceptance-test-fmt.yml

```diff
@@ -0,0 +1,15 @@
+name: Check format for acceptance tests
+
+on:
+  [pull_request]
+jobs:
+  formatting-check:
+    name: Scala formatting check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Switch to acceptance tests directory
+        run: |
+          cd acceptance-tests
+      - name: run scalafmt
+        run: |
+          sbt test:scalafmt
```

Comment:
I guess if the step suppose to be checking the formatting we should probably use `sbt scalafmtCheck` as it fails command. `sbt test:scalafmt` simply reformats the code and never fails.